src/index.ts

```diff
@@ -53,9 +53,28 @@ async function startProcessing(
   // and use the corresponding api to add the invitation email to the
   // beta group.
   unprocessedInvitations.forEach(async invitation => {
-    // TODO: Add real invitation processing
+    // Check the Invitation platform (Android/iOS)
+    const platform = invitation.platform;
 
-    // Update the invitation status in the Spreadsheet
+    // TODO: Add the logic to handle Invitations for iOS platform
+    if (platform === "iOS") {
+      return;
+    }
+
+    // Call the API to add the member to the beta group.
+    const addMemberToBetaGroupOrError = await googleClient.addMemberToBetaGroup(
+      invitation.email,
+    );
+
+    if (addMemberToBetaGroupOrError.isLeft()) {
+      logger.error(
+        `Error adding ${invitation.email} to the beta group.`,
+        addMemberToBetaGroupOrError.value,
+      );
+    }
+
+    // TODO: Add the error to the spreadsheet
```

Comment:
in case of error, will the invitation be processed again in the next loop?

Reply:
With the current code it will not be processed in the next loop. I am going to add the received error in a column of the spreadsheet. Than you can manually inspect the error in the spreadsheet and reset the PROCESSED column to reprocess the invitation in the next loop.
I think we can optimize it later to retry on temporary problems, but first i want to make the script works.